### PR TITLE
Issue 220 - added some contextual copy to Errors /docs page

### PR DESCRIPTION
**Pull request by FullStackTiger**
**What does this pull request do? Explain your changes. (required)**

**Specific updates (required)**

- Add some contextual copy to Errors /docs page

**Does this pull request close any open issues?**

- Fixes Add some contextual copy to Errors /docs page

**Screenshots (optional):**

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/www/docs/api-reference/errors.mdx [hunk 1]
```diff
@@ -12,3 +12,7 @@ The Livepeer.com API uses conventional HTTP status codes.
 `4XX` = client error, problem with information provided
 
 `5XX` = internal server error, problem with Livepeer.com servers
+
+If you encounter an error or issue that doesn't make sense or could use our
+team's attention, please let us know. Tap the chat in the lower corner of this
+page to leave us a message.
```
Comment (victorges) [inline packages/www/docs/api-reference/errors.mdx]: I think we don't have a chat in our website. WDYT of suggesting sending an email to our support instead? We have this email listed in our "Contact" page hello@livepeer.com, but maybe there is something more specific like support@livepeer.com that works?

**Review by victorges — CHANGES_REQUESTED** [review]

File: packages/www/docs/api-reference/errors.mdx (hunk 1 above)
Comment (pglowacky) [reply packages/www/docs/api-reference/errors.mdx]: We do have a chat, but it's not always staffed (sometimes, I'm sleeping 😆). Best to have people email us at hello@livepeer.com -- that is probably the fastest way to get someone's attention.

Comment (pglowacky) [reply packages/www/docs/api-reference/errors.mdx]: I'd prefer: "If you encounter an error that could use our team's attention, please email us at hello@livepeer.com"

**Review by pglowacky — CHANGES_REQUESTED** [review]

File: packages/www/docs/api-reference/errors.mdx (hunk 1 above)
Comment (victorges) [reply packages/www/docs/api-reference/errors.mdx]: Hmmm interesting, never seen it actually. Does it show up in every page? I accessed this API errors doc and didn't find any buttn for it

**Comment by FullStackTiger** [comment]
updated the text, added this mailto link for the email address @victorges

**Review by victorges — APPROVED** [review]
Awesome!

**Pull request merged**